roles/create-openshift-resources/tasks/create_persistent_volume_claims.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: "Set Basic Persistent Volume Claim Facts"
+  set_fact:
+    pvc: "{{ persistent_volume_claims_item }}"
+
+- name: Fail for Missing Persistent Volume Claim Parameters
+  fail: msg="This role requires persistent_volume_claims.{{ item }} be set and non empty"
+  when: pvc.{{ item }} is not defined or pvc.{{ item }} == ''
+  with_items:
+  - name
+  - access_modes
+  - storage
+
+- name: "Determine if {{ pvc.name }} Persistent Volume Claim Exists"
```

Comment:
The below 5  tasks seem idiomatic. I wonder if this would be a good place for a module/role to handle object creation via jinja templates. It could support create vs apply